src/deercreeklabs/baracus.cljc

```diff
@@ -1,6 +1,8 @@
 (ns deercreeklabs.baracus
   (:refer-clojure :exclude [byte-array])
   (:require
+    [clojure.string]
```

Comment:
These two namespaces are used but weren't required. This ends up being fine if something has loaded them and you refer to them by their full name (which is the case later in this file). I do remember one instance of making this assumption in my code and then running into a situation where the nobody else loaded the namespace so my code broke.